demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -217,16 +222,18 @@ class PackParser(BaseContentParser, PackMetadataParser):
 
     content_type = ContentType.PACK
 
-    def __init__(self, path: Path) -> None:
+    def __init__(self, path: Path, git_sha: Optional[str] = None) -> None:
         """Parses a pack and its content items.
 
         Args:
             path (Path): The pack path.
         """
+        if path.name == PACK_METADATA_FILENAME:
+            path = path.parent
```

Comment:
why is it needed?

Reply:
We want to be able to parse a pack & pack_metadata from 'from_path' (-i inputs) so in some cases we receive pack_metadata as an input and this is how we handle this case.